src/app/utils/Comparable.ts

```diff
@@ -24,7 +24,8 @@ function comparableEqualsStrict<T extends Comparable>(a: T, b: T): boolean {
             const bJSON: ComparableJSON = b as ComparableJSON;
             for (const key of Object.keys(aJSON)) {
                 if (key in bJSON) {
-                    if (comparableEquals(aJSON[key], bJSON[key]) === false) {
+                    // eslint-disable-next-line @typescript-eslint/no-use-before-define
```

Comment:
non c'est l'entiereté de use-before-define qui est horrible, j'ai pas trouvé de config ou il est pas débile :D HATE THIS :D

Reply:
hm ici il est nécessaire car il y a une définiton circulaire, me semble pas qu'on en ai des tonnes des comme ça ? C'quand même mieux de définir-utiliser plutôt qu'utiliser-définir tu trouve pas ? :D

Reply:
nvm, on part sur la suppression du warning donc j'enlève !

Reply:
bah, quand c'est les fonction, dans un classe, mettre d'abord une fonction suivit dans l'ordre des fonctions qu'elle a utilisé, ça fait une lecture plus sympa non ?

Reply:
yup vrai !

Reply:
Du coup on décide quoi, on supprime ce linting ?

Reply:
oui, ça a été décidé et fait dans ta dernière branche mergée non ? (ici plus d'apparence de no-use-before define)